### PR TITLE
Create vn_vthanhtivi.m3u

### LINKED ISSUE #15743: Add: History (Vietnam Feed)
### Channel ID (required)

HistoryHD.vn

### Channel Name

History Channel

### Stream URL (required)

http://fpt.vthanhtivi.pw/history/mpegts

### Quality

1080p

### Label

Geo-blocked

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

N/A

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #15744: Add: Gem
### Channel ID (required)

GEM.sg

### Channel Name

GEM

### Stream URL (required)

https://code.vthanhtivi.pw/getlink/eazie/242918/playlist.m3u8

### Quality

1080p

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

cant find channel id

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #15745: Add: HBO
### Channel ID (required)

HBOHD.vn

### Channel Name

HBO

### Stream URL (required)

http://fpt.vthanhtivi.pw/hbo/index.m3u8

### Quality

1080p

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

DONT KNOW WHICH CHANNEL ID IS FOR THIS CHANNEL
### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #15757: Add: Cinemax Asia?
### Channel ID (required)

CinemaxAsia

### Channel Name

Cinemax Asia

### Stream URL (required)

http://fpt.vthanhtivi.pw/cinemax/index.m3u8

### Quality

None

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

Don't know which id is for this channel

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #15762: Add: ONE HD (SG)
### Channel ID (required)

ONEHD.sg

### Channel Name

One HD

### Stream URL (required)

https://code.vthanhtivi.pw/getlink/eazie/243114/playlist.m3u8

### Quality

1080p

### Label

Geo-blocked

### Timeshift

0

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Dum4G**
Resolves https://github.com/iptv-org/iptv/issues/15762
Resolves https://github.com/iptv-org/iptv/issues/15757
Resolves https://github.com/iptv-org/iptv/issues/15745
Resolves https://github.com/iptv-org/iptv/issues/15744
Resolves https://github.com/iptv-org/iptv/issues/15743

### CODE REVIEW

File: streams/vn_vthanhtivi.m3u [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+#EXTINF:-1 tvg-id="ONEHD.sg",One HD
+https://code.vthanhtivi.pw/getlink/eazie/243114/playlist.m3u8
+#EXTINF:-1 tvg-id="",Cinemax Asia (Vietnam)
+http://fpt.vthanhtivi.pw/cinemax/index.m3u8
+#EXTINF:-1 tvg-id="",HBO (Vietnam)
+http://fpt.vthanhtivi.pw/hbo/index.m3u8
+#EXTINF:-1 tvg-id="GEM.sg",GEM
+https://code.vthanhtivi.pw/getlink/eazie/242918/playlist.m3u8
+#EXTINF:-1 tvg-id="",History (Vietnam)
+http://fpt.vthanhtivi.pw/history/mpegts
```
Comment (BellezaEmporium) [inline streams/vn_vthanhtivi.m3u:11]: Wouldn't it be easier to use the index.m3u8 file ? http://fpt.vthanhtivi.pw/history/index.m3u8

**Review by BellezaEmporium — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Comment by Jitterdoomer** [comment]
It's all broken as of 2/18/2024.

**Comment by Dum4G** [comment]
Cinemax at least works for me

**Comment by Jitterdoomer** [comment]
> Cinemax at least works for me

Both ONE and Gem doesn't work. Only Vietnamese airing of HBO, Cinemax, and History Channel worked.

**Comment by Jitterdoomer** [comment]
Sorry to say that both Gem and One had a prank livestream.

**Pull request merged**